dist/etc/conf/nucleo-l432kc.env

```diff
@@ -0,0 +1,84 @@
+HIL_DUT_RST_GPIO?=
+HIL_DUT_RST_PORT?=
+HIL_DUT_RST_PIN?=
+
+HIL_DUT_CTS_GPIO?=
+HIL_DUT_CTS_PORT?=
+HIL_DUT_CTS_PIN?=
+
+HIL_DUT_RTS_GPIO?=
+HIL_DUT_RTS_PORT?=
+HIL_DUT_RTS_PIN?=
+
+HIL_DUT_RX_GPIO?="PA9/CN3-2/D1"
+HIL_DUT_RX_PORT?=0
+HIL_DUT_RX_PIN?=9
+
+HIL_DUT_TX_GPIO?="PA10/CN3-1/D0"
+HIL_DUT_TX_PORT?=0
+HIL_DUT_TX_PIN?=10
+
+HIL_DUT_NSS_GPIO?="PA4/CN4-9/A3"
+HIL_DUT_NSS_PORT?=0
+HIL_DUT_NSS_PIN?=4
+
+HIL_DUT_SCK_GPIO?="PB3/CN4-15/D13"
+HIL_DUT_SCK_PORT?=1
+HIL_DUT_SCK_PIN?=3
+
+HIL_DUT_MISO_GPIO?="PB4/CN3-15/D12"
+HIL_DUT_MISO_PORT?=1
+HIL_DUT_MISO_PIN?=4
+
+HIL_DUT_MOSI_GPIO?="PB5/CN3-14/D11"
+HIL_DUT_MOSI_PORT?=1
+HIL_DUT_MOSI_PIN?=5
+
+HIL_DUT_SCL_GPIO?="PB6/CN3-8/D5"
+HIL_DUT_SCL_PORT?=1
+HIL_DUT_SCL_PIN?=6
+
+HIL_DUT_SDA_GPIO?="PB7/CN3-7/D4"
+HIL_DUT_SDA_PORT?=1
+HIL_DUT_SDA_PIN?=7
+
+HIL_DUT_PWM_GPIO?="PA8/CN3-12/D9"
+HIL_DUT_PWM_PORT?=0
+HIL_DUT_PWM_PIN?=8
+
+HIL_DUT_DAC_GPIO?="PA7/CN4-6/A6"
+HIL_DUT_DAC_PORT?=0
+HIL_DUT_DAC_PIN?=7
+
+HIL_DUT_ADC_GPIO?="PA5/CN4-8/A4"
+HIL_DUT_ADC_PORT?=0
+HIL_DUT_ADC_PIN?=5
+
+HIL_DUT_IC_GPIO?="PA6/CN4-7/A5"
+HIL_DUT_IC_PORT?=0
+HIL_DUT_IC_PIN?=6
+
+HIL_DUT_GPIO0_GPIO?="PA0/CN4-12/A0"
+HIL_DUT_GPIO0_PORT?=0
+HIL_DUT_GPIO0_PIN?=0
+
+HIL_DUT_GPIO1_GPIO?="PA1/CN4-11/A1"
+HIL_DUT_GPIO1_PORT?=0
+HIL_DUT_GPIO1_PIN?=1
+
+HIL_DUT_GPIO2_GPIO?="PA3/CN4-10/A2"
+HIL_DUT_GPIO2_PORT?=0
+HIL_DUT_GPIO2_PIN?=3
+
+HIL_UART_DEV?=1
+HIL_SPI_DEV?=0
+HIL_SPI_HW_CS?=
+HIL_I2C_DEV?=0
+HIL_AN_DEV?=
```

Comment:
Is it possible to define the adc and channel of the dut connected to the philip dac (if riot doesn't support it in the periph conf then leave it blank).

Reply:
I couldn’t find anything about in the periph conf